python/lsst/ts/externalscripts/auxtel/latiss_base_align.py

```diff
@@ -829,13 +836,37 @@ async def arun(self, checkpoint: bool = False) -> None:
                 self.offset_total_coma_y += coma_y
                 await self.atcs.offset_aos_lut(z=focus_offset, x=coma_x, y=coma_y)
 
+            # Not sure if here is the best place to get number of executed
```

Comment:
So, there's already a iterator being used to track down number of iterations. I think you can either do `self.executed_iterations = i` at the start of the loop, or maybe even switch from using `i` as the iterator to using `self.executed_iterations`, e.g.:

```
for self.executed_iterations in range(self.max_iter):
    ...
```

I think this one is a bit more elegant actually although, you would have to replace all references to `i`. 

I would also recommend renaming the variable to something like `iterations_executed`, it is always best practice to start a variable name with a noun.

Reply:
Perfect.  I changed the iterator variable (and all appearances) to `self. iterations_executed`.